app/styles/custom.css

```
@@ -166,7 +166,7 @@ a:hover {
   text-decoration-thickness: 2px;
 }
 
-h1, h2, h3, h4, h5, h6 {
+h1, h2, h3, h4, h5, h6, .table > thead > tr > th, strong, bold, .bold, .semibold {
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)